### ISSUE TITLE
Imports generation issue with raml-to-jaxrs maven plugin

### ISSUE THREAD

**Issue by amagnus** [mulesoft-labs/raml-for-jax-rs#320]
Hello,

The issue is that the generated ResponseDelegate contains this:

```
import java.lang.Class;
import java.lang.Object;
import java.lang.Override;
import java.lang.String;
import java.lang.annotation.Annotation;
```

It should only contain this java.lang import:

```
import java.lang.annotation.Annotation;
```

This is causing a conflict with maven-checkstyle-plugin which is throwing errors when simple java.lang imports are present. So we end up having to manually edit ResponseDelegate where we shouldn't touch generated code.

I looked into it and found that there is a `skipJavaLangImports` flag on the javapoet library, which can be set here:
https://github.com/mulesoft-labs/raml-for-jax-rs/blob/master/raml-to-jaxrs/jaxrs-code-generator/src/main/java/org/raml/jaxrs/generator/ResponseSupport.java#L105

However it doesn't work when I try to set it to true.

Just my two cents, as we're trying to reduce friction around the maven plugin.

**Comment by jstoiko** [comment]
Seems like we're depending on an [older version of javapoet](https://github.com/mulesoft-labs/raml-for-jax-rs/blob/3ff80e058554faede5bbdcb2ff6afa5d85a1cb53/raml-to-jaxrs/jaxrs-code-generator/pom.xml#L23). Maybe that option was introduced later on or not working as expected in that version? MY 2c :)

**Comment by amagnus** [comment]
Tested the latest version with same result.

**Comment by jpbelang** [comment]
Fixed on release/2.2.0
